### PR TITLE
blockdiag 2.0.1 (new formula)

### PR DESCRIPTION
**Pull request by rhwood**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/blockdiag.rb [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  sha256 "16a69dd9f3b44c9e0869999ce82aa968586698febc86ece9ca0c902dba772397"
+
+  depends_on "freetype"
+  depends_on "imagemagick"
+  depends_on "jpeg"
+  depends_on "libtiff"
+  depends_on "little-cms2"
+  depends_on "python"
+  depends_on "webp"
+  depends_on "zlib"
```
Comment (rhwood) [inline Formula/blockdiag.rb:16]: `brew audit --strict blockdiag` accepts depending on zlib, but `brew audit --new-formula blockdiag` requires that I use `uses_from_macos`, however this does not build if I do not depend on zlib.

Comment (rhwood) [reply Formula/blockdiag.rb:16]: Why did this `depends_on "zlib"` work on my Mac, but fail on the CI builds?

Comment (chenrui333) [reply Formula/blockdiag.rb:16]: it failed on `brew audit`, we can certainly use `uses_from_macos` one. Let me try it.

File: Formula/blockdiag.rb [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        group {
+          label = "group label";
+          fontsize = 16;  // change fontsize of group-label
+
+          C;
+        }
+      }
+    EOS
+    system "#{bin}/blockdiag", "-a", "test.diag"
+    # output is test.png, but how do I verify good png?
```
Comment (SMillerDev) [inline Formula/blockdiag.rb:64]: Is the output of the command png data? Or does it write to a file?

**Comment by rhwood** [comment]
It’s written to a file named `test.png` in the same directory as `test.diag`.

Randall

**Comment by SMillerDev** [comment]
> It’s written to a file named `test.png` in the same directory as `test.diag`. 

In that case verifying that the file is there after the test should be enough.

**Pull request closed without merging**